[2022-12-12 05:26:04,411] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-12T05:25:34.742287+00:00 [queued]>
[2022-12-12 05:26:04,527] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-12T05:25:34.742287+00:00 [queued]>
[2022-12-12 05:26:04,537] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 05:26:04,540] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 05:26:04,541] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 05:26:04,759] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-12-12 05:25:34.742287+00:00
[2022-12-12 05:26:04,811] {standard_task_runner.py:52} INFO - Started process 7778 to run task
[2022-12-12 05:26:04,890] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'manual__2022-12-12T05:25:34.742287+00:00', '--job-id', '1642', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmp4rxfgamh', '--error-file', '/tmp/tmp0jc8rmkw']
[2022-12-12 05:26:04,898] {standard_task_runner.py:80} INFO - Job 1642: Subtask world
[2022-12-12 05:26:05,395] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world manual__2022-12-12T05:25:34.742287+00:00 [running]> on host 7e39590a7807
[2022-12-12 05:26:06,484] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-12-12T05:25:34.742287+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-12T05:25:34.742287+00:00
[2022-12-12 05:26:06,526] {logging_mixin.py:115} INFO - Airflow
[2022-12-12 05:26:06,531] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 05:26:06,793] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20221212T052534, start_date=20221212T052604, end_date=20221212T052606
[2022-12-12 05:26:07,829] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 05:26:09,286] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
